extras/catch_amalgamated.cpp

```diff
@@ -6,7 +6,7 @@
 // SPDX-License-Identifier: BSL-1.0
 
 //  Catch v3.1.0
-//  Generated: 2022-07-17 20:14:05.885021
```

Comment:
You should not regenerate this on PR. It's generated only once before releasing a version by the main maintainer.

Reply:
I was not sure if the CI tests would generate it because it seems to be tested. It can easily be undone because it is a separate commit.

Reply:
Removed.